blob/fileblob/example_test.go

```diff
@@ -65,9 +66,16 @@ func Example_openBucket() {
 	}
 	defer os.RemoveAll(dir)
 
+	// On Unix, append the dir to "file://".
+	// On Windows, convert "\" to "/" and add a leading "/":
+	dirpath := filepath.ToSlash(dir)
+	if os.PathSeparator != '/' && !strings.HasPrefix(dirpath, "/") {
+		dirpath = "/" + dirpath
+	}
+
 	// OpenBucket creates a *blob.Bucket from a URL.
 	ctx := context.Background()
-	b, err := blob.OpenBucket(ctx, path.Join("file://", dir))
+	b, err := blob.OpenBucket(ctx, "file://"+dirpath)
```

Comment:
`path.Join` is the wrong thing here, as it collapses multiple slashes, resulting in URLs like `file:/tmp/dir`.

Reply:
Ack.